services/wfm/src/main/java/org/openkilda/wfm/topology/flow/validation/FlowValidator.java

```diff
@@ -113,4 +122,41 @@ void checkFlowForEndpointConflicts(Flow requestedFlow) throws FlowValidationExce
                             conflictedFlow.get().getFlowId()));
         }
     }
+
+    /**
+     * Validates switch existing.
+     *
+     * @param requestedFlow a flow to be validated.
+     * @param switches a list of all switches.
+     * @throws SwitchValidationException if switch not found.
+     */
+    @SuppressWarnings("squid:S1192")
+    public void validateSwitches(Flow requestedFlow, List<SwitchInfoData> switches) throws SwitchValidationException {
```

Comment:
Second argument is excess. Validator must be able to fetch any required info by himself.